src/styles/sidebar/components/svg-icon.scss

```diff
@@ -0,0 +1,10 @@
+/* Aids in icon alignment */
```

Comment:
Can you change the comment here to something like "Make the wrapper element's size match the `<svg>` element"?